modules/cbelasticsearch/models/mappings/MappingBlueprint.cfc

```diff
@@ -0,0 +1,231 @@
+/**
+*
+* ElasticSearch Mapping Blueprint Object
+*
+* @package cbElasticsearch.models.mappings
+* @author  Eric Peterson <eric@ortussolutions.com>
+* @license Apache v2.0 <http://www.apache.org/licenses/>
+*
+*/
+component accessors="true" {
+
+    property name="wirebox" inject="wirebox";
+    property name="builder" inject="MappingBuilder@cbElasticSearch";
+
+    property name="properties";
+
+    function init() {
+        variables.properties = [];
+        return this;
+    }
+
+    function text( name ) {
+        var mapping = variables.builder.newSimpleMapping( name, "text" );
```

Comment:
This is the only one that breaks backward compat.  Do we want to try to pull the ES version or keep it only compatible with 6.0+?